app/src/main/java/com/yeonae/chamelezone/view/home/StartActivity.kt

```diff
@@ -28,7 +38,17 @@ class StartActivity : AppCompatActivity(), StartContract.View {
         presenter = StartPresenter(
             Injection.memberRepository(App.instance.context()), this
         )
-        presenter.logout()
-
+        val setting = getSharedPreferences("setting", 0)
+        val email = setting.getString("ID", "")
+        val password = setting.getString("PW", "")
+        if (email != null && password != null) {
+            if (email.isNotEmpty() && password.isNotEmpty()) {
+                presenter.userLogin(email, password)
+            } else {
+                presenter.logout()
+            }
+        } else {
+            presenter.logout()
+        }
```

Comment:
```suggestion
        if (!email.isNullOrEmpty() && !password.isNullOrEmpty()) {
            presenter.userLogin(email, password)
        } else {
            presenter.logout()
        }
```
의미는 같으나 조금 간결하게 축약할 수 있을 것 같습니다..

Reply:
수정했습니당